src/hooks/core/chat/useMoveChatTypeHook.ts

```diff
@@ -144,10 +133,10 @@ const useMoveChatTypeHook = () => {
             anon_user_info_emoji_name: anonProfileResult?.anon_user_info_emoji_name,
             source
           }));
-
       const messages = result?.data?.messageHistories?.map((item: any) => item?.message);
       const channel = {
         ...result?.data?.channel,
+        better_channel_member: result?.data?.better_channel_members,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Two properties (`better_channel_member` and `members`) are now set to `result.data.better_channel_members`. If these two properties are meant to have the same value, consider refactoring the code to avoid redundancy. If they are supposed to be different, make sure the correct values are assigned.


<!-- This is an auto-generated comment by OSS CodeRabbit -->